packages/formstr-app/src/containers/CreateFormNew/components/QuestionsList/index.tsx

```diff
@@ -141,44 +169,59 @@ export const QuestionsList = () => {
       ref={containerRef}
       style={{ position: "relative" }}
     >
-      <div>
-        <FormTitle className="form-title" />
-        <DescriptionStyle>
-          <div className="form-description">
-            <Input.TextArea
-              key="description"
-              value={formSettings.description}
-              onChange={handleDescriptionChange}
-              autoSize
-            />
-          </div>
-        </DescriptionStyle>
+      <div style={{ padding: "10px", paddingBottom: "0px" , textAlign: "center" }}>
```

Comment:
instead of showing up randomly here, we can move this to the form header, next to the Form Builder, like "AI builder". It grounds the feature with other formstr stuff.